docs/sequence-diagrams/api-integration.puml

```diff
@@ -24,15 +24,15 @@ group golden values provisioning
 	Node -> BE: POST /node/secret, Body: {nodeID}
 	BE -> V_EviVfy: POST /newSession
 	V_EviVfy -> V_EviVfy: /session/123=>{"nonce"="the-challenge", ...}
-	V_EviVfy -> BE: Status: 201, Location: /session/123, Body: {"nonce"="the-challenge", ...}
-	BE -> BE: Encrypt(EK_pub, "the-challenge") -> N
-	BE -> Node: Status: 200, Body: {N}
+	V_EviVfy -> BE: Status: 201, Location: /session/123, Body: {challenge-blob, ...}
```

Comment:
`challenge-blob` is computed by BE in the following step.  The nonce in this response body is effectively Veraison's generated "the-challenge"


```suggestion
	V_EviVfy -> BE: Status: 201, Location: /session/123, Body: {"nonce"="the-challenge", ...}
```